e2e/config/config.go

```diff
@@ -370,6 +376,9 @@ func (c configTests) configGlobal(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		if tt.addRequirementsFn != nil {
+			tt.addRequirementsFn(t)
```

Comment:
it will skip the whole group, you must pass it to `PreRun` function: `e2e.PreRun(tt.addRequirementsFn)`